test/api/importCSV/api_test.go

```diff
@@ -0,0 +1,29 @@
+package importCSV
```

Comment:
Please name the package without capital letters, like importcsv. Check https://go.dev/doc/effective_go